packages/core/src/App/Components/Layout/Header/toggle-notifications.jsx

```diff
@@ -49,4 +65,4 @@ const ToggleNotificationsDrawer = ({
     );
 };
 
-export default ToggleNotificationsDrawer;
+export default observer(ToggleNotificationsDrawer);
```

Comment:
@sergei-deriv Wrap the component definition at line 9 with `observer` 🙏🏻

Reply:
@farzin-deriv changed